js/src/admin/components/ExtensionPermissionGrid.js

```diff
@@ -1,44 +1,38 @@
 import PermissionGrid from './PermissionGrid';
-import ItemList from '../../common/utils/ItemList';
+import ItemList from "../../common/utils/ItemList";
 
 export default class ExtensionPermissionGrid extends PermissionGrid {
+  oninit(vnode) {
+    super.oninit(vnode);
+
+    this.extensionId = this.attrs.extensionId;
+  }
+
   permissionItems() {
-    const items = super.permissionItems();
+    const permissionCategories = super.permissionItems();
 
-    Object.keys(items.items).map((item) => {
-      if (items.items[item].content.children.length === 0) {
-        items.remove([item]);
+    Object.keys(permissionCategories.items).map((item) => {
```

Comment:
I wonder if this could be done with spread syntax or `.entries()` and `Object.fromEntries()` so that we could use `fiilter`